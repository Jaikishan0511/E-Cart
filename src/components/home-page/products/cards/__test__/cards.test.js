import React from "react";
import Cards from "../index";
import ReactTestRenderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
const CardsComp = (
    <Router>
      <Cards />
    </Router>
  );
describe("Actions", () => {
    describe("Render", () => {
      it("should be a valid React Element", () => {
        expect(React.isValidElement(<Cards />)).toBeTruthy();
      });
  
      it("should render without throwing an error", () => {
        expect(() =>
          ReactTestRenderer.create(CardsComp)
        ).not.toThrow();
      });
    });
  
    describe("Snapshots", () => {
      it("matches the default snapshot", () => {
        const tree = ReactTestRenderer.create(CardsComp).toJSON();
        expect(tree).toMatchSnapshot();
      });
    });
  });