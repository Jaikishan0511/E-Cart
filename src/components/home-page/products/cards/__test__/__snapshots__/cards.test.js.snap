// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions Snapshots matches the default snapshot 1`] = `
<div
  className="card"
>
  <div
    className="image-container"
  >
    <img
      alt="logo"
      src="/images/logo.png"
    />
  </div>
  <div
    className="product-name"
  >
    <h3>
      
    </h3>
    <p>
      
    </p>
  </div>
  <div>
    <h5>
      Price
    </h5>
    <p>
       Rs 
      0
       
    </p>
    <div
      className="add-to-cart"
    >
      <button
        onClick={[Function]}
        value={0}
      >
        Add
      </button>
    </div>
  </div>
</div>
`;
